test/arrays/ferguson/loc-arr-slice.good

```diff
@@ -2,9 +2,5 @@ loc-arr-slice.chpl:1: In function 'badReturnRefLocal':
 loc-arr-slice.chpl:3: error: illegal expression to return by ref
 loc-arr-slice.chpl:6: In function 'badReturnRefConstant':
 loc-arr-slice.chpl:7: error: function cannot return constant by ref
-loc-arr-slice.chpl:19: In function 'badReturnAliasLocal':
-loc-arr-slice.chpl:21: error: illegal return of array aliasing a local array
-loc-arr-slice.chpl:31: In function 'badReturnAliasAliasLocal':
-loc-arr-slice.chpl:33: error: illegal return of array aliasing a local array
```

Comment:
It seems like, ideally, we'd like to preserve these warnings, right?  To that end, could we create a version of this test that had the old .good as a future, with a "retire this test once it's working and the original's .good is updated/restored" type of note in its .future?